data/remnant missions.txt

```diff
@@ -60,27 +62,94 @@ mission "First Contact: Remnant"
 	
 	on decline
 		event "ember waste label"
+		set "Remnant Blood Test Pure"
 		log "Discovered a strange, isolated human community in an otherwise uninhabited region of space south of the Core. They call themselves the Remnant."
 		log "Factions" "Remnant" `Half a millennium ago, during the Alpha Wars, a group of humans fleeing the conflict and looking for a safer place to live discovered a wormhole into a region of space known as the Ember Waste. They built settlements there, and chose to call themselves the Remnant. Although they no longer fear the Alphas as much as they once did, they still have not rejoined human society.`
 			`Since their colonies were first formed, the Remnant's culture has changed drastically. They communicate in sign language, using their voices only for singing. Their food and architecture and much of their technology is strange, and much of it has been borrowed or stolen from nearby alien species.`
 	
 	npc kill
 		government "Remnant"
 		personality heroic coward disables plunders
-		fleet "Large Remnant" 2
+		fleet "Large Remnant" 1
 
 event "ember waste label"
 	galaxy "label waste"
 		sprite label/waste
 
+mission "Remnant: Second Contact"
+	name "Second chance at First Contact with the Remnant"
+	landing
+	invisible
+	source
+		government "Remnant"
+	description "A Remnant provides you with the opportunity to take a second run at first contact."
+	clearance "You can't understand the video feed of gestures, but your ship's computer receives a landing grid solution via antique protocols."
+	destination Viminal
+	waypoint Segesta
+	to offer
+		not "Remnant Blood Test Pure"
+		not "Remnant Second Chance Refused"
+	to fail
+		or
+			has "Remnant Second Chance Refused"
+			has "Remnant Blood Test Pure"
+	on enter Segesta
+		conversation
+			`After your escape from Remnant space, you are uneasy at how quickly you lost your pursuers. The eerie red light of the Ember Wastes sets an ominous background for the sensation that you are being watched. Time and time again you check the long range scanners, but there is absolutely nothing out there.`
+			`	Just as  you were starting to relax, your comms ding with an incoming ship-to-ship transmission. When you hit the 'accept' key a melodic song bursts forth from your speakers.`
+			`	"My compatriots seem to have gotten off on the wrong foot. I am prefect Chilia of the Remnant. We have established ourselves here as a bastion of humanity to survive when when the rest of human space was under threat by the Alphas."`
```

Comment:
I was intending this conversation to be audio-only, hence the reference to "your speakers" and just describing the audio side. I could change this if people think it would better. It always strikes me as odd when a cloaked ship starts broadcasting video, though. Not impossible or anything, just incongruant.

Title is corrected, and I removed one of the "have"s. The quotation marks have also been corrected.